### DISCUSSION TITLE
Your profile is under review, looks like you have not filled the complete form and thats causing some delay. We request you to kindly update the information if not done.

### DISCUSSION THREAD

**Discussion by kartikmehta8** [github/india#678] [Students]
Anybody else facing the same issue?

**Comment by vivaan19** [comment]
Same issue with me also, please resolve it

**Reply by code-23** [reply]
I am also facing the same issue

**Reply by pra17dod** [reply]
same here!!

**Reply by ayushsonone07** [reply]
Same here

**Reply by ayushigeorge** [reply]
same here! what to do next?

**Reply by raaghavrm** [reply]
did any one got the solution ?

**Comment by llneonll** [comment]
> Anybody else facing the same issue?

I'm also facing the same issue, already filled all fields

**Reply by SusmitaSen681** [reply]
I am also facing same issue

**Reply by highonweb** [reply]
I have the same issue. is it resolved for anyone?

**Reply by harshitarora-in** [reply]
I am also facing the same issue.

**Reply by raaghavrm** [reply]
did anyone got the solution ??

**Comment by SusmitaSen681** [comment]
No it's not resolved

**Reply by codevardhan** [reply]
has it been resolved yet? Mine's been stuck there for some time as well

**Comment by Pavanyeluri11** [comment]
Is the issue solved for anyone?

**Reply by raaghavrm** [reply]
did you got the solution ?

**Comment by Gayatribhosale2610** [comment]
Yes,  same issue with me also.

**Comment by 0xRitesh** [comment]
I'm still having the same problem. Is anyone's problem solved?

**Reply by Ayush7614** [reply]
same issue

**Reply by raaghavrm** [reply]
did you guys got the solution ?

**Comment by raaghavrm** [comment]
Did anyone got a solution ??

**Reply by raaghavrm** [reply]
Please reply
